ReleaseNotes-2.9

```diff
@@ -58,6 +58,12 @@ Wanted changes:
 
  * The editing of a user's realname, email adress or password is no longer possible if LDAP mode is activated
 
+ * Unused ldap options in options.yml were dropped:
```

Comment:
@hennevogel nope. Our changes are not the cause of this. The options are simply just not used in the whole code.